assets/js/steps/utils/bibbox-sounds.js

```diff
@@ -0,0 +1,191 @@
+/**
```

Comment:
I don't think this is the way to solve this. 
It seems too much bookkeeping code, for something that should just be an event listener that handles events, e.g.

```
let playing = false;

.on('playSound', function (sound) {
  let file = null;
  switch (sound) {
    case 'birthday':
      file = 'birthday.mp3';
  }
 
  if (file !== null && !playing) {
    const audio = new Audio(file);
    audio.play();
    audio.on('ended', function () {
      playing = false;
    });
    playing = true;
  }
});
```

The bookkeeping of whether or not the sound should be played should be handled by the component that wants to play a sound. The sound component should not have to know everything about the rest of the system.

I don't see why a react component is needed to play a sound.